keyboards/mechwild/sugarglider/config.h

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 Kyle McCreery
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "config_common.h"
+
+#ifdef VIA_ENABLE
+#define DYNAMIC_KEYMAP_LAYER_COUNT 5
+#endif
+
+/* Matrix COL and ROW definitions */
+#define MATRIX_ROWS 9
+#define MATRIX_COLS 6
+
+/* Cirque Touchpad Settings */
+#define POINTING_DEVICE_AUTO_MOUSE_ENABLE
+#define USB_POLLING_INTERVAL_MS 1
+#define CIRQUE_PINNACLE_ATTENUATION EXTREG__TRACK_ADCCONFIG__ADC_ATTENUATE_2X
+
+/* Memory definitions for UF2 builds */
+#ifdef UF2_BUILD
+#define EXTERNAL_EEPROM_BYTE_COUNT 2048
+#define EXTERNAL_EEPROM_PAGE_SIZE 128
+#define EXTERNAL_EEPROM_ADDRESS_SIZE 1
+#define EXTERNAL_EEPROM_WRITE_TIME 0
+#define FEE_PAGE_BASE_ADDRESS 0x08008000
+#endif
```

Comment:
There are a couple of issues with this block. 

the FEE page is only for emulated eeprom, the rest is for external eeprom.  Ed, you'd need one or the other. 

Additionally, both the f401and f411 support emulated eeprom when using the tinyuf2 bootloader. 

Eg, this entire block is not necessary.
```suggestion
```